.changeset/sortable-chips-field-improvements.md

```diff
@@ -0,0 +1,8 @@
+---
+"saleor-dashboard": patch
+---
+
+Enhanced SortableChipsField component with collapsible functionality and internationalization:
+- Added "Show less" button to collapse expanded chips list back to showing only the first 5 chips
+- Implemented proper internationalization for button text using react-intl
```

Comment:
suggestion: I don't think we need to mention that, it's a part of this feature :)

```suggestion
```

Reply:
I will clean this up in the next PR